tst/testinstall/opers/LatticeByCyclicExtension.tst

```diff
@@ -5,14 +5,18 @@ gap> G:=SmallGroup(500,1);
 <pc group of size 500 with 5 generators>
 gap> fun:=g->IsInt(4/Size(g));;
 gap> LatticeByCyclicExtension(G);
-<subgroup lattice of <pc group of size 500 with 5 generators>, 12 classes, 
-164 subgroups>
+<subgroup lattice of <pc group of size 
+500 with 5 generators>, 12 classes, 164 subgroups>
 gap> LatticeByCyclicExtension(G, fun);
-<subgroup lattice of <pc group of size 500 with 5 generators>, 3 classes, 
-127 subgroups, restricted under further condition l!.func>
+<subgroup lattice of <pc group of size 
```

Comment:
@fingolfin Any idea why my changes are now causing everything in this test file to appear differently? The new way seems much worse. Any idea how to get back to the old behaviour?

Reply:
It took me some time to figure this one out. But in the end, the problem is that `Print("123")` and `Print(123)` do something different:
1. The former calls the kernel function `PrintString1` which just prints the string:
```C
void PrintString1(Obj list)
{
    Pr("%g", (Int)list, 0);
}
```
2. The latter goes through `PrintObj` and finally to `PrintInt`, which does this:
```C
void PrintInt ( Obj op )
{
  /* print a small integer                                                 */
  if ( IS_INTOBJ(op) ) {
    Pr("%>%d%<", INT_INTOBJ(op), 0);
  }
...
```
The symbols `%<` and `%>` give hints to GAP where to (not) perform line breaks.

So the simplest fix would be to not produce the string `"123 generators"` but rather to produce the string `"%>123%< generators"`. Of course that might *not* be what one wants in all cases; in that case, one could introduce an option to turn off generation of these strings... but I don't think we have need for that right now, so I think it'd be OK to just insert those line break markers?

Reply:
I've added the hints \> and \< to the returned string, and incorporated that. So hopefully the tests will pass and this can be merged 🙂